client/lib/analytics/utils.js

```diff
@@ -86,6 +88,19 @@ export function hashPii( data ) {
 		.digest( 'hex' );
 }
 
+/**
+ * Returns the current user email after normalizing it (lowercase without spaces) or `false` if no email or user is available.
+ * @return {false|string} The current user email after normalization
+ */
+export function getNormalizedHashedUserEamil() {
+	const currentUser = user.get();
+	if ( currentUser && currentUser.email ) {
+		hashPii( currentUser.email.toLowerCase().replace( /\s/g, '' ) );
```

Comment:
Missing `return` statement here.

Reply:
Thanks!!! 🙏🙏🙏 Not my best PR so far...

Reply:
PR looks great to me! Little things like that happen to us all as we work through different approaches. Can't wait to run this through a full round of tests soon.